### DISCUSSION TITLE
How to decorate personal page with dynamic data like a pro?

### DISCUSSION THREAD

**Discussion by Chicken2404** [community/community#133294] [New to GitHub] [answered]
Hi everyone! I’m Brian , a new member of the GitHub community. I’m excited to connect and collaborate with you all. How to decorate personal page with dynamic data like a pro?

**Comment by 0xLou1s** [comment, marked as answer]
check here bro : https://github.com/abhisheknaiidu/awesome-github-profile-readme
write this by markdown syntax

**Reply by Chicken2404** [reply]
tks bro

**Reply by Baluprince347** [reply]
New
